Corrections typos, indentation et traductions

yapper-git opened the pull request:
| Q | R |
| --- | --- |
| Correction de bugs ? | oui |
| Nouvelle Fonctionnalité ? | non |
| Tickets (_issues_) concernés | N/A |

Les CGU contiennent actuellement trois liens incorrects.
1. Oubli mailto pour l'adresse mail communication@zestedesavoir.com (dans _2. Mentions légales_)
2. Lien vers les résultats de Google au lieu de la page pour le lien vers la CNIL (dans _6.4 Politique relative à l'installation de cookies_)
3. Inversion entre le texte du lien et l'URL pour Progdupeupl (dans _7.2 Code source du site_)

La page de profil contient une typo dans l'ancre, en effet le lien "Derniers articles" ne fonctionne pas.
De plus il y a plusieurs typos et fautes d'orthographe qui ont été répéré et corrigé par ce commit.

Est-il nécessaire que je créé une issue pour cette PR (correction de typos) ?
# QA
- Vérifier que les pages (/pages/*) s'affichent correctement.
- Vérifier que le lien (ancre) vers "Derniers articles" fonctionne correctement sur la page de profil d'un membre

yapper-git committed:
Corrections de 3 liens incorrects dans les CGU

1. Oubli mailto pour l'adresse mail
2. Lien vers les résultats de Google au lieu de la page pour le lien vers la CNIL
3. Inversion entre le texte du lien et l'URL pour Progdupeupl

SpaceFox commented:
Non, pas la peine de créer une issue :)

yapper-git commented:
Je viens de repérer d'autres erreurs sur d'autres pages, il serait certainement judicieux de les ajouter à ce commit minimaliste, je vais faire une nouvelle PR si c'est bon (et si j'y arrive aussi...).

pierre-24 commented:
Deux solutions: tu crée un second commit et tu renomme ta PR "correction orthographiques". Ou bien tu fais une seconde PR, comme tu préfères :)

yapper-git committed:
Corrections typos et indentation dans templates/pages

yapper-git committed:
Corrections des chaînes à traduire dans templates/pages
1. Inclusion de la ponctuation dans les chaînes
2. Utilisation de variables (et module set si besoin) lorsque nécessaire

yapper-git commented:
Voilà je viens de modifier la PR pour ajouter d'autres corrections aux fichiers.
1. J'ai utilisé des variables lorsque nécessaire pour fournir une chaîne correcte au lieu de bidouiller (par exemple "Adhérer à l'association {{ site_name }}" au lieu de "Adhérer à ") 
2. J'ai aussi inclus la ponctuation dans les chaînes à traduire (en effet, par exemple en anglais on ne met pas d'espace avant ! ou ? contrairement au français…)
3. Correction indentation incorrecte et d'oublis d'espaces (par exemple `{{site_name}}` au lieu de `{{ site_name }}`)

Enfin, je ne sais pas comment modifier la ligne ci-dessous pour inclure le lien dans le blocktrans et fournir l'URL dans une variable donc je n'ai pas corrigé.

```
                    <td><a href="{% url "member-detail" alert.comment.author.username %}">{{ alert.comment.author.username }}</a>, {% blocktrans with timing=alert.comment.pubdate|format_date%} posté le {{ timing }} {% endblocktrans %}</td>
```

pierre-24 commented:
[Comme tu peux le voir ici](https://github.com/Situphen/zds-site/blob/improve-front-end-doc/doc/source/front-end/template-tags.rst), on a un peu étendu ce que le langage avait à nous proposer pour palier à ce genre de chose. Du coup, je propose:

```
{% load captureas %}
{% captureas member_detail_url %}
{% url "member-detail" alert.comment.author.username %}
{% endcaptureas%}

{%blocktrans with ... %}[...] {{ member_detail_url }} {% enblocktrans %}
```

pierre-24 commented:
D'ailleurs, faudrait pas appliquer les filtres de `email_obfuscator` (même page d'info ?)

Situphen commented:
@pierre-24 on ne peut pas faire carrément `{% url "member-detail" alert.comment.author.username as member_detail_url %}` ?

pierre-24 commented:
... Si tu le dit :p

yapper-git commented:
J'ai fait des recherches et d'après la [doc officielle](https://docs.djangoproject.com/en/1.7/topics/i18n/translation/) ils proposent tout simplement d'utiliser {% url ... as var_name %} comme suggéré par @Situphen.
Du coup j'ai fait la correction et j'ai rebase et inclus la correction dans le dernier commit.

Cela donne:

```
{% url "member-detail" alert.comment.author.username as url_member_detail %}
{% blocktrans with author_username=alert.comment.author.username timing=alert.comment.pubdate|format_date %}
     <a href="{{ url_member_detail }}">{{ author_username }}</a>,  posté le {{ timing }}
{% endblocktrans %}
```

J'ai aussi remarqué que les CGU ne sont pas "traduisibles" actuellement (aucun _trans_ ou _blocktrans_ dans la vue), j'imagine que cela est volontaire.

yapper-git committed:
Correction typo dans ancre

yapper-git committed:
Correction de fautes d'orthographes

yapper-git commented:
J'ai ajouté des corrections que j'ai repéré en naviguant sur le site et en explorant le code source.

Je ne pense pas trouvé dans l'immédiat de nouvelles corrections à ajouter.

yapper-git commented:
J'ai utilisé le module set lorsque cela est utile (dans le `{% block content %}`) et rebasé.

pierre-24 commented:
Rapport de QA : c'est bon pour moi :)
